.eslintrc.js

```diff
@@ -1,20 +1,22 @@
 module.exports = {
 	extends: ["eslint:recommended"],
+	plugins: ["prettier"],
 	env: {
 		node: true,
 		es6: true,
 		jest: true
 	},
 	parserOptions: { ecmaVersion: 2017, sourceType: "module" },
 	rules: {
+		"prettier/prettier": "error",
 		"no-useless-escape": "off",
 		"quote-props": ["error", "as-needed"],
 		"no-dupe-keys": "error",
 		quotes: ["error", "double"],
 		"no-undef": "error",
 		"no-extra-semi": "error",
 		semi: "error",
-		"no-template-curly-in-string": "error",
+		"no-tempate-curly-in-string": "error",
```

Comment:
`template`?

Reply:
My bad, thanks for the quick fix @evenstensberg ✨